templates/core/crowdfunding_donors.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'global/base.html' %}
+{% load staticfiles rev %}
+
+{% block extra_css %}
+    <link rel="stylesheet" type="text/css" href="{% static_rev 'css/donors.css' %}">
+{% endblock %}
+
+{% block content %}
+<div class="row">
+    <div class="col-md-1"></div>
+    <div class="col-md-10">
+    <h2>Special thanks to all our amazing supporters!</h2>
+        <div class="row">
+        <div class="col-md-12">
+            <p>Thank you for supporting our crowdfunding campaign. Your support means the world to us.</p>
+        </div>
+        </div>
+
+        <div class="row">
+            <h2>They Support Us</h2>
+            {% for donor in donor_list %}
+                 {% if forloop.first %}<div class="row">{% endif %}
+                <div class="col-md-4">
+                    <h4 class="donor-text">
+                        <img src="{% static 'img/global/donate/pixel-heart.png' %}"> {{ donor.name }}</h4>
+                </div>
+                {% if forloop.counter|divisibleby:3 %}</div><div class="row">{% endif %}
+                {% if forloop.last %}</div>{% endif %}
+                {% if forloop.counter|divisibleby:6 %}</div><div class="row"><br />
+                    <div class="col-md-9">
+                     <blockquote>{% for quote in quotes %}{{ quote.quote }} <p>{{ quote.name }}</p>
```

Comment:
We should move `{{ quote.name }}` to the `cite` in `blockquote` as specified here: https://www.w3schools.com/tags/tag_blockquote.asp

So it will look like:
```
<blockquote cite='{{ quote.name }}'>{% for quote in quotes %}{{ quote.quote }}</blockquote>
```